src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -1194,6 +1213,8 @@ CREATE TABLE user_account
 ,description                 CHARACTER VARYING
 ,user_state                  INTEGER

```

Comment:
Wondering should this column be `BOOLEAN` like you have introduced in security_group and security_role tables.
Assuming it would make it easier to understand - is enabled TRUE etc?

Reply:
the legacy data has four acceptable values for user_state, analysis would need to be performed on the 3 states that would effectively be merged to boolean FALSE, given we have one legacy user_state that permits access.
Also, if the change to a BOOL should be acceptable, the column name should change to user_access or something similar, as user_state = TRUE, seems a little unintuitive.